.github/workflows/release.yml

```diff
@@ -39,8 +39,8 @@ jobs:
         run: |
           REF=${RELEASE_TAG:-${GITHUB_REF}}
           TAG=${REF##*/}
-          echo ::set-output name=ref::${REF}
-          echo ::set-output name=tag::${TAG}
+          echo "ref=${REF}" >> $GITHUB_OUTPUT
```

Comment:
Can you help me understand what `GITHUB_OUTPUT` is? Is it a custom environment variable? I only see `GITHUB_ENV` listed in [the documentation](https://docs.github.com/en/actions/learn-github-actions/environment-variables).

Reply:
@bejado Thanks for your comment!
The documentation for `GITHUB_OUTPUT` is a little further down that page here.
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

Reply:
This page is [linked](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) within the #6194 link It is located a little further down the page.